lib/dune

```
@@ -3,7 +3,7 @@
 
 
 (menhir (modules PPCParser) (flags  --fixed-exception))
-(menhir (modules CParser) (flags  --fixed-exception))
+(menhir (modules CParser) (flags  --fixed-exception --explain))
```

This look like leftover code:
```suggestion
(menhir (modules CParser) (flags  --fixed-exception))
```